plugins/decklink/decklink-device-instance.hpp

```diff
@@ -19,7 +19,6 @@ class DeckLinkDeviceInstance : public IDeckLinkInputCallback {
 	uint64_t                lastVideoTS = 0;
 	AudioRepacker           *audioRepacker = nullptr;
 	speaker_layout          channelFormat = SPEAKERS_STEREO;
-
```

Comment:
I wouldn't remove this line.  If you feel you must remove a blank line near here, I'd remove the one after it, and even then it doesn't do much good.